include/trainer_ids.h

```diff
@@ -2,6 +2,11 @@
 #define GUARD_TRAINER_IDS_H
 
 #define NO_OF_TRAINERS      854
+
+#define TRAINER_ID_SIDNEY   261
```

Comment:
Can you please match the defines that are in constants/trainer_constants.inc? All of those are defined in there already, and you just need to convert that .inc file into a C header. It will make things a whole lot easier in the near future when I make the .s files use the C preprocessor.

Reply:
Are the defines in trainer constants inc correct? If so, then sure, but if they're copied from Ruby I'd rather not do it, it's boring work. :P

Reply:
Actually, these were created before pokeruby existed, so they should be correct. Just use a regex to automate things and make it easy.